packages/react/src/components/Tabs/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+class Tabs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeIndex: null,
+    };
+  }
+
+  handleTabClick = activeIndex => {
+    const { buildReturnObject, onClick, elements } = this.props;
+    this.setState({ activeIndex });
+    if (onClick) {
+      onClick(
+        buildReturnObject
+          ? buildReturnObject(elements[activeIndex])
+          : activeIndex
+      );
+    }
+  };
+
+  render() {
+    const { elements, buildTabLabel } = this.props;
+    const { activeIndex } = this.state;
+
+    return (
+      <div className="Tabs">
+        {elements.map((element, index) => (
+          <span key={element} onClick={() => this.handleTabClick(index)}>
+            <div className="Tabs-label">
+              {buildTabLabel ? buildTabLabel(element) : element}
+            </div>
+            <div
+              className={cx("Tabs-marker", {
+                "is-active": activeIndex === index,
+              })}
+            />
+          </span>
+        ))}
+      </div>
+    );
+  }
+}
+
+Tabs.propTypes = {
+  /** An array of strings or objects that will make tabs content */
+  elements: PropTypes.arrayOf(
+    PropTypes.oneOfType([PropTypes.string, PropTypes.object])
```

Comment:
What about `PropTypes.node`?

Reply:
IIRC there's a linter rule about avoiding `PropTypes.object` in favor of `PropTypes.shape`
Could you update it?

```
PropTypes.shape({
  foo: PropTypes.something
  bar: PropTypes.something
})
```